elastic/kibana
# [ResponseOps][Alerting] A redundant summary alert is shown on rule details alerts list

Resolves https://github.com/elastic/kibana/issues/149824

## Summary

Added a filter to filter out logs that have an action set to `execute-action` in the `alert_summary` route on the rules details 
page.
Updated how we log summary alerts by removing the `alertId` and `group` fields. I added a new summary field to `kibana.alerting` that includes the counts for new, ongoing, and recovered alerts.

### Checklist



### To verify

- Create a rule that has Alert Summary and let the rule run.
- Click on the rule to navigate to the rule details page and look at the `Alerts` tab. Verify that the summary alert is not listed in the table.

**This is a screen shot of the table in the issue:**
We want to verify that the last row is no longer there.
<img width="1728" alt="Screen Shot 2023-02-01 at 3 02 21 PM" src="https://user-images.githubusercontent.com/109488926/216150854-590e084d-3519-4785-8375-00a7455c319f.png">
